common/src/main/java/com/microsoft/identity/common/internal/telemetry/rules/TelemetryPiiOiiRules.java

```diff
@@ -53,9 +53,9 @@
             Key.REDIRECT_URI,
             Key.HTTP_PATH,
             Key.AUTHORITY,
-            Key.IDP,
-            Key.APPLICATION_NAME,
-            Key.APPLICATION_VERSION
+            Key.IDP_NAME,
+            //App.NAME,
+            //App.VERSION
     };
 
     private TelemetryPiiOiiRules() {
```

Comment:
Minor complaint about `isPiiOrOii()` impl: rather than duplicate the logic out of `isPii` and `isOii` I'd prefer to see:

```java
    public boolean isPiiOrOii (final String propertyName) {
        return isPii(propertyName) || isOii(propertyName);
    }
```

Reply:
Addressed